java-tests/extensions/build.gradle.kts

```diff
@@ -24,6 +24,18 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+buildscript {
+    forceCodegenPlugins()
+}
+
 dependencies {
     implementation(project(":java"))
 }
+
+modelCompiler {
+    java {
+        codegen {
+            rejections().enabled.set(false)
```

Comment:
Let's leave a note to our descendants here :)

E.g.
```
The generation of rejections is disabled, 
since we don't want other plugins (potentially, 
still not using the latest ProtoData API) 
interfere with the tests of Validation ProtoData plugin.
```